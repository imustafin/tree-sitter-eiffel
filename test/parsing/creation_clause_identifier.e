class A
create
f
end
