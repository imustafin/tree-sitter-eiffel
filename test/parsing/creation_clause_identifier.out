(source_file [0, 0] - [4, 0]
  (class_declaration [0, 0] - [3, 3]
    (class_name [0, 6] - [0, 7])
    (creation_clause [1, 0] - [2, 1]
      (identifier [2, 0] - [2, 1]))))
